### PR TITLE
windows: simplify handling of gettext

### PR DESCRIPTION
**Pull request by shuffle2**
fixes binplace on first build

### CODE REVIEW

File: Source/Core/DolphinQt/Translation.cpp [hunk 1]
```diff
@@ -275,7 +275,7 @@ static bool TryInstallTranslator(const QString& exact_language_code)
     std::string lang = qlang.toStdString();
     auto filename =
 #if defined _WIN32
-        File::GetExeDirectory() + StringFromFormat("/Languages/%s/dolphin-emu.mo", lang.c_str())
+        File::GetExeDirectory() + StringFromFormat("/Languages/%s.mo", lang.c_str())
```
Comment (JosJuice) [inline Source/Core/DolphinQt/Translation.cpp:278]: Was this change done for any particular reason?

**Review by JosJuice — COMMENTED** [review]
I don't fully understand the MSBuild details, but this seems reasonable overall.

File: Source/Core/DolphinQt/Translation.cpp (hunk 1 above)
Comment (shuffle2) [reply Source/Core/DolphinQt/Translation.cpp:278]: Not really, just that I thought removing a level of directories was a clean up

Comment (JosJuice) [reply Source/Core/DolphinQt/Translation.cpp:278]: I think the intention of the old style of paths is that you could have multiple mo files per language for different components of the software... Which we never ended up using and probably won't have any reason to use in the future either, so I suppose either way is fine here.

Comment (shuffle2) [reply Source/Core/DolphinQt/Translation.cpp:278]: Yea that was my thinking too

**Review by lioncash — APPROVED** [review]

**Pull request merged**